src/components/shared/Message.js

```diff
@@ -30,12 +30,11 @@ const MessageWrapper = styled.div`
   padding: 4%;
   font-size: 13px;
   border-radius: 4px;
-  color:  ${props => props.theme.colors.primary};
+  color: ${props => props.theme.colors.primary};
   display: flex;
   align-items: center;
-  ${props => props.response && response()}
-  ${props => props.completed && completed()}
-  ${props => props.rejected && rejected()}
+  ${props => props.response && response()} ${props =>
```

Comment:
It might help to write the `response` function (and similar functions) as an arrow function, like so:
```javascipt
const response = () => css``
```